src/cookbook/forms/focus.md

```diff
@@ -49,8 +49,10 @@ First, we'll need to create a [`FocusNode`](https://docs.flutter.io/flutter/widg
 We will use the `FocusNode` to identify a specific `TextField` in Flutter's
 "focus tree." This will allow us to focus the `TextField` in the next steps.
 
-Since focus nodes are long-lived objects, we need to store them in a `State`
-class. In addition, we need to `dispose` of them when they're no longer needed!
+Since focus nodes are long-lived objects, we need to manage the lifecycle 
+using a `State`class. To do so, create the `FocusNode` instance inside the 
+`initState` method of a `State` class, and clean them up inside the `dispose` 
```

Comment:
Per writing guidelines, method names should be followed by `()`, e.g. `dispose()` method.